### PR TITLE
Try windows

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
@conda-forge/scikit-learn, do you know if `liblapack` is needed for sklearn? Is it just `libcblas`?

**Comment by isuruf** [comment]
Ready for a review. (You can ignore Azure windows failures)
This PR uses numpy from conda-forge and scipy from deaults on windows.

**Comment by isuruf** [comment]
Ping.

**Comment by rth** [comment]
> do you know if liblapack is needed for sklearn? Is it just libcblas?

Only BLAS is needed (LAPACK is used indirectly via scipy).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,16 +26,16 @@ requirements:
     - pip
     - python
     - cython
+    - blas * netlib  # [unix]
+    - libblas
+    - libcblas
```
Comment (rth) [inline recipe/meta.yaml]: Why both BLAS and BLAS?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -27,16 +26,16 @@ requirements:
     - pip
     - python
     - cython
+    - blas * netlib  # [unix]
```
Comment (rth) [inline recipe/meta.yaml]: Does this mean the reference netlib BLAS? Why can't we use OpebBLAS as was the case before (at least on Linux)?

Comment (isuruf) [reply recipe/meta.yaml]: Packages are built against netlib BLAS, but any implementation can be used at install time. See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas#blas

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml]: Fixed

**Comment by jnothman** [comment]
What is the reason for the failures?

**Comment by isuruf** [comment]
@jnothman, random MSVC compiler crashes. Restarting helps. Appveyor on windows passes, but azure doesn't.

**Pull request merged**